recipes-app/camera-setup/files/load_camera_config.sh

```diff
@@ -26,29 +26,38 @@ esac
 # will prevent the module probe to fail when loading the module after start up.
 #
 # The GPIO[11] is linked the XCZU3EG's IO_L12N_AD0N_26 pin. In the FPGA design the pin IO_L12N_AD0N_26
-# is linked the output #1 of the axi gpio IP.
-# Since the PL gpio module starts at 504, we deasser the GPIO 505 (504 + 1)
+# is linked the output #1 of the axi_gpio_0 core at address 0xa0030000.
 #
-# root@ultra96v2-2020-1:~# cat /sys/class/gpio/gpiochip504/label
-# /amba_pl@0/gpio@a0040000
+# root@u96v2-sbc-dualcam-2021-1:~# cat /sys/class/gpio/gpiochip500/label
+# a0030000.gpio
+# root@u96v2-sbc-dualcam-2021-1:~# cat /sys/class/gpio/gpiochip500/ngpio
+# 8
+source /usr/local/bin/gpio/gpio_common.sh
+
+BASE_A0030000=$(get_gpiochip_base a0030000)
```

Comment:
Is it still needed to de-assert this GPIO?
I though it was done because something was wrongfully using this GPIO (an I2C module maybe?), and that this is fixed now

Reply:
The GPIO needs to be 0, otherwise the i2c communication with the ap1302 will fail ... 
I'd rather force a known condition, rather than be subject to the behavior of the base design or other designs in the future.